core/src/dsnp/api_types.rs

```diff
@@ -0,0 +1,96 @@
+#![allow(dead_code)] // todo: remove after usage
+use crate::dsnp::{dsnp_types::DsnpId, encryption::EncryptionBehavior};
+
+/// Raw page of Graph (or Key) data
+pub type Page = Vec<u8>;
```

Comment:
nit: I think it would be better if we call it something else like `PageBlob` or something like that since we are going to have another `Page` struct for in memory page representation.

Reply:
I agree; I think some of this will evolve with the subsequent issues & we'll adjust.